<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scan_image</class>
 <widget class="QWidget" name="scan_image">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScanImage</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Original image</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButton_settings">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Settings</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ImageLabel" name="label_full_image">
        <property name="text">
         <string>Original image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_video_control">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButton_pause">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Pause</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_restart_video">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Restart video</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Board View</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="BoardView" name="board_view" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_memorizer">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Game record</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_clear_memorizer">
        <property name="text">
         <string>Clear memorizer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_memorizer_status">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Memorizer status</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KifRecorder" name="kif_recorder" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageLabel</class>
   <extends>QLabel</extends>
   <header>GUI/widgets/ImageLabel.h</header>
   <slots>
    <signal>clicked()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>BoardView</class>
   <extends>QWidget</extends>
   <header>GUI/widgets/BoardView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KifRecorder</class>
   <extends>QWidget</extends>
   <header>GUI/widgets/KifRecorder.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_pause</sender>
   <signal>clicked()</signal>
   <receiver>scan_image</receiver>
   <slot>on_pause_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>534</y>
    </hint>
    <hint type="destinationlabel">
     <x>9</x>
     <y>556</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_restart_video</sender>
   <signal>clicked()</signal>
   <receiver>scan_image</receiver>
   <slot>on_restart_video_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>534</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>533</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_settings</sender>
   <signal>clicked()</signal>
   <receiver>scan_image</receiver>
   <slot>on_settings_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_clear_memorizer</sender>
   <signal>clicked()</signal>
   <receiver>scan_image</receiver>
   <slot>on_clear_memorizer_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>910</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>941</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_pause_clicked()</slot>
  <slot>on_restart_video_clicked()</slot>
  <slot>on_settings_clicked()</slot>
  <slot>on_clear_memorizer_clicked()</slot>
 </slots>
</ui>
